<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>713</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>288</width>
    <height>713</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;跳一跳辅助程序Ver1.3.3&lt;/p&gt;&lt;p&gt;自动适配所有分辨率16：9的手机&lt;/p&gt;&lt;p&gt;Qt5.6.3&lt;/p&gt;&lt;p&gt;Opencv3.4.0&lt;/p&gt;&lt;p&gt;ADBShell&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/dengxinggao/OpencvQtTiaoyiTiaoHelp&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/dengxinggao/OpencvQtTiaoyiTiaoHelp&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>自动偏差</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开启后，会根据目标大小，决定是否多跳半个人距离&lt;/p&gt;&lt;p&gt;修改后立即生效&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>开启偏差</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_LenMultiple">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>倍率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_LenMultiple">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;可以适当修改&lt;/p&gt;&lt;p&gt;修改时，不开启偏差&lt;/p&gt;&lt;p&gt;跳的太远，调小该值&lt;/p&gt;&lt;p&gt;跳的太近，调大该值&lt;/p&gt;&lt;p&gt;修改后，再根据需要 决定是否开启偏差&lt;/p&gt;&lt;p&gt;修改后立即生效&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_connect">
        <property name="text">
         <string>连接手机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>开始跳</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>不跳了</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
